bapfs/Content/index.md

```diff
@@ -0,0 +1,6 @@
+---
+
+---
+## We Play Games
+
+We offer [Pathfinder and Starfinder Society](https://paizo.com/organizedplay) tabletop games at a variety of locations in the San Francisco Bay Area. New players are always welcome. Organized Play memberships are free — just contact us or ask one of the Event Coordinators before any game for some help. 
```

Comment:
Can/should we link to the Paizo site to get a number? Otherwise, link to the contact page?